nautobot/core/api/views.py

```diff
@@ -940,3 +941,16 @@ def get(self, request):
         else:
             data = bound_field.as_widget()
         return Response(data)
+
+
+class SettingsJSONFileView(NautobotAPIVersionMixin, APIView):
+    """View that exposes the JSON Schema of the settings.json file in the REST API"""
+
+    permission_classes = [IsAuthenticated]
+
+    @extend_schema(exclude=True)
+    def get(self, request):
+        file_path = "settings.json"
+        with open(file_path, "r") as jsonfile:
```

Comment:
This should probably be a path relative to `__file__` rather than an un-anchored path.